__tests__/utils.test.js

```diff
@@ -372,3 +352,68 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("Should update a specific article and return the articles update data", () => {
+    return request(app)
+      .patch("/api/articles/6")
+      .send({ inc_votes: 5 })
+      .expect(200)
+      .then((response) => {
+        const data = response.body.updatedArticle[0];
+        expect(data.article_id).toBe(6);
+        expect(data.hasOwnProperty("title")).toBe(true);
+        expect(data.hasOwnProperty("topic")).toBe(true);
+        expect(data.hasOwnProperty("author")).toBe(true);
+        expect(data.hasOwnProperty("body")).toBe(true);
+        expect(data.hasOwnProperty("created_at")).toBe(true);
+        expect(data.hasOwnProperty("votes")).toBe(true);
```

Comment:
It would be great to assert what the votes are after the patch. This test doesn't tell me that it has worked, just that an article comes back.